bots/s3_only_allow_ssl.py

```diff
@@ -0,0 +1,178 @@
+"""
+## s3_only_allow_ssl
+What it does: Ensure that S3 Buckets enforce encryption of data transfers using Secure Sockets Layer (SSL)
+Usage: AUTO: s3_only_allow_ssl
+Note: The bot looks at the bucket policy and adds to the current policy the missing actions(s3:GetObject and s3:PutObject)
+      and the SSL statement.
+      if no policy in the bucket an SSL policy will add to the bucket
+Limitations: none
+"""
+
+import json
+from botocore.exceptions import ClientError
+
+
+def run_action(boto_session, rule, entity, params):
+    # create a s3 sessions
+    s3_client = boto_session.client('s3')
+
+    bucket_name = entity['name']
```

Comment:
Try to use entity id

Reply:
change it to entity['id'] but in dome9 full entity id is the same as name